Observer/Customer/Save.php

```diff
@@ -40,20 +40,24 @@
 use Magento\Framework\Event\ObserverInterface;
 use Magento\Framework\Module\Manager as ModuleManager;
 use Nosto\Tagging\Helper\Data as NostoHelperData;
-use Nosto\Tagging\CustomerData\HashedTagging;
+use Nosto\Tagging\Helper\Customer as NostoHelperCustomer;
 
 class Save implements ObserverInterface
 {
     private $moduleManger;
+    private $nostoHelperCustomer;
 
     /**
      * Save constructor.
      * @param ModuleManager $moduleManger
+     * @param NostoHelperCustomer $nostoHelperCustomer
      */
     public function __construct(
-        ModuleManager $moduleManger
+        ModuleManager $moduleManger,
+        NostoHelperCustomer $nostoHelperCustomer
```

Comment:
This injection will become redundant after generation is placed into a static util